ci.sh

```diff
@@ -267,14 +267,10 @@ else
         netsh winsock reset
     fi
 
-    # Disable coverage on 3.8 until we run 3.8 on Windows CI too
-    #   https://github.com/python-trio/trio/pull/784#issuecomment-446438407
-    if [[ "$(python -V)" = Python\ 3.8* ]]; then
-        true;
     # coverage is broken in pypy3 7.1.1, but is fixed in nightly and should be
     # fixed in the next release after 7.1.1.
     # See: https://bitbucket.org/pypy/pypy/issues/2943/
```

Comment:
I guess 7.2.0 is out, so possibly we can remove this too...

Reply:
I want to try that next. 👍